packages/atlas-service/src/store/atlas-signin-reducer.spec.ts

```diff
@@ -176,10 +177,7 @@ describe('atlasSignInReducer', function () {
 
       void store.dispatch(performSignInAttempt()).catch(() => {});
 
-      await Promise.all([
-        store.dispatch(signIn()),
-        store.dispatch(cancelSignIn()),
-      ]);
```

Comment:
We are already attempting signin in `performSignInAttempt` and doing it here, triggers `isAuthenticated` stub and hence rejecting the promise after the test has finished with its assertions. 

So I removed extra signin and also ensured that `isAuthenticated` is only called once.